spec/latest/common/terms.html

```diff
@@ -212,6 +212,14 @@
       A <a>prefix</a> is a <a>term</a> that expands to a vocabulary <a>base IRI</a>. It
       is typically used along with a <em>suffix</em> to form a <a>compact IRI</a> to create an IRI
       within a vocabulary.</dd>
+    <dt><dfn>processing mode</dfn></dt><dd>
+      The processing mode defines how a JSON-LD document is processed.
+      By default, all documents are assumed to be conformat with
+      <a data-cite="JSON-LD-20140116">JSON-LD 1.0</a> [[!JSON-LD-20140116]]. By defining
+      a different version using the <code>@version</code> member in a
```

Comment:
That thought occurred to me, but it would be more natural, then, to say `"@processingMode": "json-ld-1.1"`, which would pass a 1.0 processor silently.

An alternative suggested by @lanthaler was to use something like `"@processingMode": ["json-ld-1.1"]`, which would error by a 1.0 processor, would would allow the symmetry with the API mode. Personally, I preferred using the simpler numeric value,  but I could be convinced to go the other way.